src/main/python/covata/delta/apiclient.py

```diff
@@ -229,7 +225,7 @@ def get_secret(self, requestor_id, secret_id):
         :param str requestor_id: the authenticating identity id
         :param str secret_id: the secret id to be retrieved
         :return: the retrieved secret
-        :rtype: dict[str, any]
+        :rtype: dict[str, str]
```

Comment:
here, can't remember exactly but I'm pretty sure there's also a nested `dict` of encryption details in the response json. same case with identity response json.